TYPO3.Neos/Classes/TYPO3/Neos/Utility/BackendAssetsUtility.php

```diff
@@ -39,7 +39,7 @@ public function injectSettings(array $settings)
      */
     public function shouldLoadMinifiedJavascript()
     {
-        return isset($this->settings['userInterface']['loadMinifiedJavaScript']) ? $this->settings['userInterface']['loadMinifiedJavaScript'] : $this->settings['userInterface']['loadMinifiedJavascript'];
+        return isset($this->settings['userInterface']['loadMinifiedJavaScript']) ? $this->settings['userInterface']['loadMinifiedJavaScript'] : true;
```

Comment:
makes sense I guess although I'd consider something is wrong is that setting is unavailable for some reason.. but this change breaks something else.. notice the subtle difference in spelling.. first it checks for `JavaScript` and if not found `Javascript` since it was often misspelled.. so you'd need to account for that and only if both aren't set, return `true`